pool/web/handler/public.ml

```diff
@@ -107,6 +107,29 @@ let not_found req =
   result |> Http_utils.extract_happy_path req
 ;;
 
+let denied req =
```

Comment:
Shouldn't the denied handler just return a hardcoded friendly error message in English and `403`?